src/test/scala/edu/ie3/simona/agent/grid/ThermalGridIT.scala

```diff
@@ -735,7 +736,7 @@ class ThermalGridIT
             }
         }
 
-      scheduler.expectMessage(Completion(heatPumpAgent, Some(71891)))
+      scheduler.expectMessage(Completion(heatPumpAgent, Some(71892)))
```

Comment:
And here, tick is even increased instead of decreased...

Reply:
Right. I followed this backwards:

1. one tick difference in variable duration in `ThermalHouse.nextActivation` caused by 
2. slightly different qDot (8e-6) and slightly different `flexibleEnergy` (what a name) (6e-4), the latter caused by
3. different innerTemp because, my assumption, we calculate at ticks that are a little bit different and round there also a little bit. 

imho this is ok, but yes, over time this error might add up a little bit. Which it always has been, right? Or am I missing something?